app/pages/admin/organization-status.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import moment from 'moment';
+
+import ProjectIcon from '../../components/project-icon';
+import LoadingIndicator from '../../components/loading-indicator';
+
+import Toggle from './project-status/toggle';
+
+class OrganizationStatus extends Component {
+  constructor(props) {
+    super(props);
+    this.forceUpdate = this.forceUpdate.bind(this);
```

Comment:
The bad reasoning for code as is was duplicating the current project admin components, but I'll rewrite these organization admin components to use `componentWillReceiveProps` and also refactor the project admin components in a separate PR.